js/tsconfig.generated.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
How do you feel about having comments in this json file? I think typescript will allow it.
It would be nice to have a comment here about the purpose of this file.

Reply:
Yeah, TypeScript has its own JSON parser. 👍